Code/Framework/AzNetworking/AzNetworking/Utilities/EncryptionCommon.cpp

```diff
@@ -100,19 +100,35 @@ namespace AzNetworking
 
     static void GetCertificatePaths(TrustZone trustZone, AZStd::string& certificatePath, AZStd::string& privateKeyPath)
     {
-        const AZ::CVarFixedString certificateFile = (trustZone == TrustZone::ExternalClientToServer) ? net_SslExternalCertificateFile : net_SslInternalCertificateFile;
-        const AZ::CVarFixedString privateKeyFile = (trustZone == TrustZone::ExternalClientToServer) ? net_SslExternalPrivateKeyFile : net_SslInternalPrivateKeyFile;
-
-        AZStd::string assetDir;
-        if (AZ::IO::FileIOBase::GetInstance() != nullptr)
+        if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
         {
-            char buffer[AZ_MAX_PATH_LEN];
-            AZ::IO::FileIOBase::GetInstance()->ResolvePath("@products@/", buffer, sizeof(buffer));
-            assetDir = AZStd::string(buffer);
-        }
+            AZ::CVarFixedString certificateFile = "";
+            AZ::CVarFixedString privateKeyFile = "";
+            if (trustZone == TrustZone::ExternalClientToServer)
+            {
+                console->GetCvarValue("net_SslExternalCertificateFile", certificateFile);
+                console->GetCvarValue("net_SslExternalPrivateKeyFile", privateKeyFile);
```

Comment:
Is there a reason you're using console->GetCvarValue vs using something similar to the pre-existing ternary? If you need console->GetCvarValue to access it safely, that might imply a bug with accessing the variable.

Reply:
Discussed offline and this relates to https://github.com/o3de/o3de/issues/5537

Reply:
Do we plan to create a GHI for changing console->GetCvarValue back to the pre-existing ternary once #5537 is fixed? These changes seem like workaround for an existing bug but we may want to clean it up later.

Reply:
no plan, and if necessary that should be included within existing GHI. It doesn't make sense to create a separate GHI for this work.